test/meson.build

```diff
@@ -44,6 +44,7 @@ extra_test_sources = files([
 
 test_env = environment()
 test_env.set('MESON_BUILD_DIR', meson.build_root())
+test_env.set('OPENSSL_CONF', join_paths(meson.build_root(), '..', 'test', 'openssl-pkcs11.cnf'))
```

Comment:
This should be configured in `test/conftest.py`, as only the pytest-bases test-cases use PKCS#11 and can also be run without meson.